### PR TITLE
Information about the current view

### PR DESCRIPTION
**Pull request by brauliorivas**
based on #61 
BEGINRELEASENOTES
- According to #54, every view should have information about it. Explain how the relations work, what colors mean, etc. So now, if a user opens the information modal, may switch to the `view` section, and learn more. 

ENDRELEASENOTES
Right now, it is shown simple information about the view. We can explain more about each view, so its more useful.

### CODE REVIEW

File: js/views/views-dictionary.js [hunk 1]
```diff
@@ -25,65 +25,82 @@ export const views = {
     viewFunction: mcParticleTree,
     scrollFunction: scrollTopCenter,
     preFilterFunction: preFilterMCTree,
+    description:
+      "Red relations mean parent relation (from bottom to top), green relations mean daughter relation (from top to bottom).",
```
Comment (tmadlener) [inline js/views/views-dictionary.js:29]: Can we do html in the description for styling?

**Review by tmadlener — COMMENTED** [review]
Nice. One thing I would do is to make the `This View` the default and make `About eede` the thing to get to via a click. Simply because information about the view is more "useful" than about the viewer in general.

**Comment by kjvbrt** [comment]
Can the button for switching of the view information be hidden when the view is not used? I mean whn the user is at the intro modal.

**Comment by kjvbrt** [comment]
Also, whoudl it be possible to not have "This view", but the name of the view?

**Review by tmadlener — COMMENTED** [review]
The result looks good to me. It's a bit hard to check with the example file that we have for full CLD simulation because at least for me some of the views take quite long to load. But I see you have already started to investigate in #63

**Comment by tmadlener** [comment]
> Also, whoudl it be possible to not have "This view", but the name of the view?

Oh yes, that would be perfect and even better than my "This view" suggestion.

**Comment by kjvbrt** [comment]
This looks "good to go" for me :)

**Pull request merged**